apps/test/netsim/NetSimRouterNode.js

```diff
@@ -30,6 +32,54 @@ describe("NetSimRouterNode", function () {
     testShard = fakeShard();
   });
 
+  describe("default row structure", function () {
+    var row;
+
+    beforeEach(function () {
+      var router = new NetSimRouterNode(testShard);
+      row = router.buildRow_();
+    });
+
+    it ("dnsMode (default DnsMode.NONE)", function () {
+      assertOwnProperty(row, 'dnsMode');
+      assertEqual(row.dnsMode, DnsMode.NONE);
+    });
+
+    it ("dnsNodeID (default undefined)", function () {
+      assertOwnProperty(row, 'dnsNodeID');
+      assertEqual(row.dnsNodeID, undefined);
+    });
+
+    it ("bandwidth (default 'Infinity')", function () {
+      assertOwnProperty(row, 'bandwidth');
+      assertEqual(row.bandwidth, 'Infinity');
+    });
```

Comment:
What's the argument for breaking this into three different its? I'd have only done it in one. Not saying my way is better, but curious about the thinking.